providers/os/connection/snapshot/blockdevices.go

```diff
@@ -112,7 +112,7 @@ func (blockEntries blockDevices) GetUnmountedBlockEntry() (*fsInfo, error) {
 }
 
 func (entry blockDevice) IsNoBootVolume() bool {
-	return entry.Uuid != "" && entry.FsType != "" && entry.FsType != "vfat" && entry.Label != "EFI"
+	return entry.Uuid != "" && entry.FsType != "" && entry.FsType != "vfat" && entry.Label != "EFI" && entry.Label != "boot"
```

Comment:
this is the reason my tests kept failing so much - i was testing against a rhel machine and i was picking up the boot volume, which we would then of course fail to mount and everything would go a little haywire